### PR TITLE
Alert avatar

### PR DESCRIPTION
**Pull request by archit7448**
For avatar and alert

### CODE REVIEW

**Review by sanu29 — APPROVED** [review]
_Good. 
But will suggest some changes. 

When resetting is done don't use that property again as it is automatically inherited.
eg: Box-sizing property in your code

Naming conventions should be same and meaningful
eg alert-Alert This should not be used__

**These could be changed later also as per your convince so you are good to go.**

**Comment by archit7448** [comment]
Thank you for the suggestion updated

File: Components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    font-size: 1.6rem;
+    margin-left: 5px;
+}
+
+.alerts-para{
+    font-size: 1.6rem;
+    font-weight: 300;
+    margin-left: 5px;
+}
+.alert-cross{
```
Comment (zoef-yt) [inline Components/alert/alert.css:21]: spacing

File: Components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    background-color: white;
+    margin-left: auto;
+}
+.alert i{
```
Comment (zoef-yt) [inline Components/alert/alert.css:25]: spacing

File: Components/avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
+.avatar-img{
+    width: 100%;
+    height: 100%;
+    border-radius: 50%;
+    object-fit: cover;
+}
+.avatar-xlg{
```
Comment (zoef-yt) [inline Components/avatar/avatar.css:7]: spacing

File: Components/avatar/avatar.css [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+.avatar-sml{
+     width: 4rem;
+     height: 4rem;
+}
+
+.avatar-xsml{
+    width:2rem;
+    height: 2rem;
+}
+.avatar-img{
```
Comment (zoef-yt) [inline Components/avatar/avatar.css:27]: spacing

**Review by zoef-yt — CHANGES_REQUESTED** [review]
There is some inconsistency in spacing. make it consistent throughout the code.

File: Components/avatar/avatar.css (hunk 4 above)
Comment (archit7448) [reply Components/avatar/avatar.css:27]: Updated thank you for the suugestion

File: Components/alert/alert.css [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    color: #4D90Fd;;
+}
+
+
+.alert-notification i{
+    color: #FC9700;
+}
+
+.alert-error i{
+    color:red;
```
Comment (varun13169) [inline Components/alert/alert.css]: user color variables.

For Dark theme you might want to use darker shade of red. Changing at 10 different places in project is not good idea.

**Review by varun13169 — CHANGES_REQUESTED** [review]

File: Components/alert/alert.css (hunk 5 above)
Comment (archit7448) [reply Components/alert/alert.css]: Updated thank you for the suggestion

**Review by varun13169 — APPROVED** [review]

**Pull request merged**